hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -141,6 +143,8 @@ public class SCMNodeManager implements NodeManager {
   private final String opeState = "OPSTATE";
   private final String comState = "COMSTATE";
   private final String lastHeartbeat = "LASTHEARTBEAT";
+  private final String usedSpacePercent = "USEDSPACEPERCENT";
+  private final String totalCapacity = "CAPACITY";
```

Comment:
These should be constants.  (The existing ones, too, but those are being fixed in HDDS-10152.)

Reply:
@adoroszlai , do we need to wait for HDDS-10152 to be resolved?

Reply:
No, I don't think we need to wait.

(However, I'd like to get HDDS-10157 fixed before merging anything else, since CI is failing in native check.)